lib/sles4sap.pm

```diff
@@ -372,7 +372,8 @@ sub prepare_profile {
 
     if ($has_saptune) {
         assert_script_run 'saptune service takeover';
-        my $ret = script_run("saptune solution verify $profile", die_on_timeout => 0);
+        enter_cmd "saptune solution verify $profile; echo DONE-$\? > /dev/$serialdev";
```

Comment:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19908 is merged

Reply:
yes. I already rebased on top.